lib/travis/build/script/c.rb

```diff
@@ -3,7 +3,7 @@ module Build
     class Script
       class C < Script
         DEFAULTS = {
-          compiler: 'gcc'
+          compiler: ''
```

Comment:
I don't think this is correct.

When `compiler` is not given in `.travis.yml`, the announcement will be ` --version`, which results in "`command not found`" (though not critical), and the cache slug will lack this information (also not critical).